fiware-region-sanity-tests/commons/neutron_operations.py

```diff
@@ -120,6 +129,53 @@ def create_router(self, router_name, external_network_id=None):
         self.logger.debug("Created router %s", neutron_network_response['router']['id'])
         return neutron_network_response['router']
 
+    def add_router_interface(self, router_id, subnetwork_id):
```

Comment:
I'd suggest renaming this method to "add_interface_router", for coherence with Nova Client (in other cases, names coincide)